scripts/import_evals_cache.py

```
@@ -0,0 +1,87 @@
+            errors += 1
+            continue
+
+        target = cache_dir / filename
+        if target.exists() and not force:
+            print(f"  skip (exists): {target}", file=sys.stderr)
+            skipped += 1
+            continue
+
+        try:
+            data = base64.b64decode(b64_data)
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)